pipenv/cli.py

```diff
@@ -1020,12 +1022,12 @@ def check(three=None, python=False):
     # Assert each specified requirement.
     for marker, specifier in p.data['_meta']['requires'].items():
 
-            if marker in results:
-                try:
-                    assert results[marker] == specifier
-                except AssertionError:
-                    failed = True
-                    click.echo('Specifier {0} does not match {1} ({2}).'.format(crayons.green(marker), crayons.blue(specifier), crayons.red(results[marker])))
+        if marker in results:
```

Comment:
Cool! For future PRs, we like to keep things pretty tightly coupled to the issue at hand. I would usually ask this be split out into it's own PR, but the change is done and pretty straight forward so we'll leave it here this time :)